client/dashboard/data/domains.ts

```diff
@@ -64,3 +65,11 @@ export async function fetchDomainSuggestions(
 
 	return suggestions;
 }
+
+export async function updateDNSSEC( domain: string, enabled: boolean ) {
```

Comment:
Could we add the response type (Promise) like in https://github.com/Automattic/wp-calypso/blob/d66260106bc743329f02a3c2f79e5a50dca0eb70/client/dashboard/data/domains.ts#L33-L39

That helps a lot with type inference. Thanks!